beacon_node/store/src/lib.rs

```diff
@@ -175,6 +178,9 @@ pub enum DBColumn {
     BeaconBlock,
     #[strum(serialize = "blb")]
     BeaconBlob,
+    /// Block roots of orphaned beacon blobs.
+    #[strum(serialize = "blbo")]
+    BeaconBlobOrphan,
```

Comment:
Usually we keep these to 3 bytes, it probably doesn't matter _too much_ but the extra byte would get stored on disk for every key.

```suggestion
    #[strum(serialize = "blo")]
    BeaconBlobOrphan,
```